### PR TITLE
Rendering different Distance Transforms

### PR DESCRIPTION
**Pull request by anne-gropler**
Added a lot of GUI elements (see Issue #36 ).
Fixes path management.
Includes options for windows-installer creation.

### CODE REVIEW

File: source/llassetgen/include/llassetgen/llassetgen.h [hunk 1]
```diff
@@ -5,8 +5,7 @@
 #include "Atlas.h"
 #include "DistanceTransform.h"
 #include "FntWriter.h"
-#include "FontFinder.h"
-#include "Packing.h"
+#include "FontFinder.h"#include "Packing.h"
```
Comment (scheibel) [inline source/llassetgen/include/llassetgen/llassetgen.h]: This seems strange.

File: source/llassetgen/source/Image.cpp [hunk 2]
```diff
@@ -234,8 +234,7 @@ namespace llassetgen {
                 memcpy(&data[(min.y + y) * stride + min.x / 8],
                        &ft_bitmap.buffer[y * ft_bitmap.pitch],
                        rowLength);
-            }
-        } else {
+            }        } else {
```
Comment (scheibel) [inline source/llassetgen/source/Image.cpp]: I'd argue its hard to track nesting here.

File: source/llassetgen/include/llassetgen/llassetgen.h (hunk 1 above)
Comment (anne-gropler) [reply source/llassetgen/include/llassetgen/llassetgen.h]: that's definitly a merge result, i'll fix this.

File: source/llassetgen/source/Image.cpp (hunk 2 above)
Comment (anne-gropler) [reply source/llassetgen/source/Image.cpp]: same as above. sometimes, when I merge files, newlines get deleted.

**Comment by anne-gropler** [comment]
@scheibel you (partly) reviewed. Would you approve or continue reviewing?
@Felerius @Lichtso @Blaidd-Drwg @j-tr Would you please review/approve?
@cgcostume Would you review, too, please?

File: source/llassetgen/CMakeLists.txt [hunk 3]
```diff
@@ -46,8 +46,7 @@ set(headers
     ${include_path}/Image.h
     ${include_path}/DistanceTransform.h
     ${include_path}/FntWriter.h
-    ${include_path}/FontFinder.h
-    ${include_path}/Geometry.h
+    ${include_path}/FontFinder.h    ${include_path}/Geometry.h
```
Comment (cgcostume) [inline source/llassetgen/CMakeLists.txt]: minor glitch?

File: source/llassetgen/CMakeLists.txt [hunk 4]
```diff
@@ -56,8 +55,7 @@ set(sources
     ${source_path}/Image.cpp
     ${source_path}/DistanceTransform.cpp
     ${source_path}/FntWriter.cpp
-    ${source_path}/FontFinder.cpp
-    ${source_path}/packing/internal/Common.cpp
+    ${source_path}/FontFinder.cpp    ${source_path}/packing/internal/Common.cpp
```
Comment (cgcostume) [inline source/llassetgen/CMakeLists.txt]: same here

**Review by cgcostume — APPROVED** [review]

**Pull request merged**